ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -481,30 +481,30 @@ def check_model_consistency(logger):
         :type logger: ovs.log.healthcheck_logHandler.HCLogHandler
         """
 
-        logger.info("Checking model consistency: ", 'check_model_consistency')
+        logger.info('Checking model consistency: ', 'check_model_consistency')
 
         # RabbitMQ check: cluster verification
-        logger.info("Pre-check: verification of RabbitMQ cluster: ", 'checkRabbitMQcluster')
-        if Helper.get_ovs_type() == "MASTER":
-            r = RabbitMQ(ip=OpenvStorageHealthCheck.MACHINE_DETAILS.ip)
+        logger.info('Pre-check: verification of RabbitMQ cluster: ', 'checkRabbitMQcluster')
+        if Helper.get_ovs_type() == 'MASTER':
+            r = RabbitMQ(ip=OpenvStorageHealthCheck.LOCAL_SR.ip)
             partitions = r.partition_status()
             if len(partitions) == 0:
-                logger.success("RabbitMQ has no partition issues!",
+                logger.success('RabbitMQ has no partition issues!',
                                'process_rabbitmq')
             else:
-                logger.failure("RabbitMQ has partition issues: {0}".format(', '.join(partitions)),
+                logger.failure('RabbitMQ has partition issues: {0}'.format(', '.join(partitions)),
                                'process_rabbitmq')
         else:
-            logger.skip("RabbitMQ is not running/active on this server!",
+            logger.skip('RabbitMQ is not running/active on this server!',
                         'process_rabbitmq')
 
         #
         # Checking consistency of volumedriver vs. ovsdb and backwards
         #
 
         for vp in VPoolHelper.get_vpools():
-            if vp.guid in OpenvStorageHealthCheck.MACHINE_DETAILS.vpools_guids:
-                logger.info("Checking consistency of volumedriver vs. ovsdb for vPool '{0}': ".format(vp.name))
+            if vp.guid in OpenvStorageHealthCheck.LOCAL_SR.vpools_guids:
```

Comment:
This is actually not a check which should be executed on every node, but on only 1 node, since this check is clusterwide